### PR TITLE
icds-cas pillow & celery configurations

### PR DESCRIPTION
**Pull request by emord**
@dimagi/scale-team 

Being fairly conservative here. We should still have plenty of RAM and CPU after this. main changes are form partitions 16 -> 24 and case partitions 36 -> 60. 

Going to start a doc about how I think scaling to 250k works with some current code changes that are being worked on.

### CODE REVIEW

File: environments/icds-cas/app-processes.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ celery_processes:
   'celery1':
     ucr_queue:
       concurrency: 4
-      max_tasks_per_child: 5
+      max_tasks_per_child: 1
```
Comment (snopoke) [inline environments/icds-cas/app-processes.yml]: why this change?

**Comment by emord** [comment]
doc as mentioned: https://docs.google.com/document/d/1ET9Gm24CuuYlLvuWhcrhyjkHlKp6EY-nsgLui6Oww1Y/edit#

File: environments/icds-cas/app-processes.yml (hunk 1 above)
Comment (emord) [reply environments/icds-cas/app-processes.yml]: It's not a necessary change, but left a comment about why in the commit https://github.com/dimagi/commcare-cloud/pull/2062/commits/1c7ee7d78407bea740303610bd0fe2e4a635b858

> There isn't a lot of activity on these queues and it's likely a high risk for memory leaks due to loading many documents

**Comment by snopoke** [comment]
@emord I'll let you roll out these changes

**Comment by emord** [comment]
already deployed
